api/src/identity-access-management/application/api/users-api.js

```diff
@@ -6,3 +6,7 @@ import { usecases } from '../../domain/usecases/index.js';
 export const markLevelSevenInfoAsSeen = async ({ userId }) => {
   return usecases.rememberUserHasSeenLevelSevenInformation({ userId });
 };
+
+export const updateHasSeenNewDashboardInfo = async ({ userId }) => {
+  return usecases.updateUserHasSeenNewDashboardInfo({ userId });
+};
```

Comment:
Est-ce qu'on peut renommer en `markNewDashboardInfoAsSeen` pour avoir une API plus orienté métier que BDD. Et être consistent avec l'autre méthode de l'API.
```suggestion
export const markNewDashboardInfoAsSeen = async ({ userId }) => {
  return usecases.updateUserHasSeenNewDashboardInfo({ userId });
};
```